shared/database/source_control/working_dir.py

```diff
@@ -278,13 +279,27 @@ def verify_directory_in_project(session, project, directory_id):
         Other ways we could do this but this seems to work as a basic check
         Future perhaps permissions on individual directories
         """
-        directory = session.query(WorkingDir).filter(
-            WorkingDir.project_id == project.id,
-            WorkingDir.id == directory_id).first()
-        if directory:
+        
+        directory = WorkingDir.get(session, directory_id, project.id)
+        return bool(directory)
```

Comment:
Small cleanup of the method above, since it was duplicate code from the `WorkingDir.get` method